test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
```

Comment:
This should be a regular console app:

Microsoft.NET.Sdk not Microsoft.NET.Sdk.Web